### PR TITLE
Show data usage summary

### PR DESCRIPTION
**Pull request by ashinzekene**
I've been using this for quite a while now. Just thought to create a PR for it. I put in an image on the README to show a sample usage

### CODE REVIEW

**Comment by mykeels** [comment]
Hey, this is cool. 

It doesn't show up when I visit [your branch](https://github.com/ashinzekene/spectranet-cli/tree/ashinzekene/usage-summary)

image

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 This is an unofficial CLI tool for accessing Spectranet user account data from its web portal.
 
-![Spectranet CLI Usage Preview](https://user-images.githubusercontent.com/11996508/46118338-f10f5280-c1fd-11e8-800d-53a55002b9d1.png)
+![Spectranet CLI Daily Usage Preview](./spectranet-daily-usage.PNG)
```
Comment (mykeels) [inline README.md:5]:
```suggestion
![Spectranet CLI Daily Usage Preview](https://user-images.githubusercontent.com/11996508/85010676-50db2900-b158-11ea-8a74-38fc1549dac5.png)
```

File: README.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
...
+![Spectranet CLI Usage Preview](./spectranet-usage.PNG)
```
Comment (mykeels) [inline README.md:6]:
```suggestion
![Spectranet CLI Usage Preview](https://user-images.githubusercontent.com/11996508/85010688-56d10a00-b158-11ea-8ca1-271783249e33.png)
```

**Comment by mykeels** [comment]
Using my commit suggestions should fix the problem, and if you remove the binary files from the PR, I'm happy to merge.

**Comment by mykeels** [comment]
Actually, one more thing.

Could you indicate the commands that give the functionality for both of the screenshots, beginning with `spectranet-cli`?

E.g.

- `spectranet-cli usage`

**show screenshot**

- `spectranet-cli daily-usage`

**show screenshot**

**Comment by ashinzekene** [comment]
Okay, I'd put in the suggested change.

Also, for the screenshots, it's only the command `spectranet-cli usage` that shows the logs

**Comment by mykeels** [comment]
How does it differentiate between showing the progress bar and not doing so?

**Comment by ashinzekene** [comment]
Here is the full image

image

**Comment by mykeels** [comment]
Oh awesome! A screenshot like that, showing the command for it would work then.

**Comment by ashinzekene** [comment]
Yeah. I think it's fixed now

**Review by mykeels — APPROVED** [review]

**Comment by mykeels** [comment]
Thank you!!!

**Pull request merged**